src/main/java/org/mockito/internal/junit/JUnitTool.java

```diff
@@ -4,15 +4,32 @@
  */
 package org.mockito.internal.junit;
 
+import org.mockito.exceptions.verification.ArgumentsAreDifferent;
+
 public class JUnitTool {
 
-    private static final JUnitDetecter detecter = new JUnitDetecter();
+    private final static boolean hasJUnit = canCreateJUnitClass();
+
+    private JUnitTool() {
+    }
+
+    public static AssertionError createArgumentsAreDifferentException(String message, String wanted, String actual) {
+        if (hasJUnit()) {
+            return new org.mockito.exceptions.verification.junit.ArgumentsAreDifferent(message, wanted, actual);
+        }
+        return new ArgumentsAreDifferent(message);
+    }
 
     public static boolean hasJUnit() {
```

Comment:
Is this method used somewhere else? Else we can just remove it and then on line 17 just remove the method invocation.